Makefile

```diff
@@ -1,6 +1,12 @@
 PHP_CS_FIXER_FUTURE_MODE=1
 PHPSTAN=./phpstan.phar
 PHP-CS-FIXER=./php-cs-fixer-v2.phar
+PHPUNIT=vendor/bin/phpunit
+
+DOCKER_RUN=docker run -t --rm -e PHPDBG=1 -v "$$PWD":/opt/infection -w /opt/infection
```

Comment:
Why do we need `-e PHPDBG=1` here?

Reply:
Changing `-it` to `-t` allows parallel execution.

Reply:
` -e PHPDBG=1` causes the e2e tests to execute with PHPDBG